config/initializers/json_web_token.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class JsonWebToken
```

Comment:
it's a service - not blyat, and initializer

Initializer adds configurations and setting for gems, not define methods and classes and all these sheet.